.github/workflows/pr-main_l2.yaml

```diff
@@ -45,10 +45,16 @@ jobs:
           name: buildx-cache
           path: /tmp/buildx-cache
 
-  integration-test:
-    # "Integration Test" is a required check, don't change the name
-    name: Integration Test
+  test:
```

Comment:
you don't have unit tests? Because generally we have "Test" for unit tests and "Integration Test" for integration tests. Both are required checks

Reply:
No we don't, I'll rollback the name change.